RadixWallet/Features/AssetsFeature/Components/StakeUnitList/Components/StakeClaimNFTsView.swift

```diff
@@ -1,26 +1,12 @@
-// MARK: - StakeClaimNFTSView
-public struct StakeClaimNFTSView: View {
+// MARK: - StakeClaimResourceView
+public struct StakeClaimResourceView: View {
 	public struct ViewState: Sendable, Hashable {
-		public let canClaimTokens: Bool
 		public let validatorName: String?
-		public let stakeClaimTokens: OnLedgerEntitiesClient.NonFunbileResourceWithTokens
-
-		var selectedStakeClaims: IdentifiedArrayOf<NonFungibleGlobalId>?
-
-		var unstaking: IdentifiedArrayOf<OnLedgerEntitiesClient.StakeClaim> {
-			stakeClaimTokens.stakeClaims.filter(\.isUnstaking)
-		}
-
-		var readyToBeClaimed: IdentifiedArrayOf<OnLedgerEntitiesClient.StakeClaim> {
-			stakeClaimTokens.stakeClaims.filter(\.isReadyToBeClaimed)
-		}
-
-		var toBeClaimed: IdentifiedArrayOf<OnLedgerEntitiesClient.StakeClaim> {
-			stakeClaimTokens.stakeClaims.filter(\.isToBeClaimed)
-		}
+		public var stakeClaimTokens: StakeClaimTokensView.ViewState
```

Comment:
I suppose this should actually be a let?

Reply:
It is var because it is needed to update the selected stake claims array in selection mode.

Reply:
Ah, so we are reusing the ViewState type as State? That would also explain the identified arrays. But for such cases I wonder if we shouldn't do it the other way around, and declare it as a State, and reuse it as ViewState, rather than the other way around. We are not consistent about that, but I think it makes more sense to see the State as primary. Something to put on the todo list.